modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -0,0 +1,3 @@
+.diagram-fixed-row {
+    height: 230px !important;
```

Comment:
Do we need `!important`? It should be used as a last resort, to overwrite inline JS, added by third party JS. If you're just overriding simple CSS, add it along with with `.diagram-fixed-row`. For example, if the style is on `.row`, add `.diagram-fixed-row.row {...}`